RecoParticleFlow/PFTracking/interface/PFTrackAlgoTools.h

```diff
@@ -18,5 +18,6 @@ namespace PFTrackAlgoTools {
                         const std::vector<double>& DPtovPtCut,
                         const std::vector<unsigned>& NHitCut,
                         bool useIterTracking,
-                        bool debug = false);
+                        bool debug = false,
+			std::string trackQuality = "highPurity");
```

Comment:
I would suggest to swap the order, and have `trackQuality` before `debug`.

Reply:
Also
```suggestion
			const std::string& trackQuality = "highPurity");
```

Reply:
The change of order would complicate backward compatibility and introduce changes to where this function ("goodPtResolution") is used without trackQuality variable as input explicitly.

Let me know if this is ok (in principle I agree that having "debug" as last variable).

As far as I can tell, the affected classes are these ones:
RecoParticleFlow/PFTracking/plugins/HGCalTrackCollectionProducer.cc
RecoParticleFlow/PFProducer/plugins/importers/GeneralTracksImporterWithVeto.cc

Reply:
@makortel is your suggested change ready to go into the commit or needs testing?
@fwyzard let me know if the other classes should be changed to accommodate the suggested change of order